kafka-observer/run.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -euo pipefail
+
+BASEDIR="$(dirname "$0")"
+
+# Resources vars
+CM_TOPICJOBS_NAME="topic-to-jobs"
+AUTH_SECRETS_NAME="authentication-secrets"
+
+# Consumers vars
+CONSUMER_JOB_NAME_1="payments-consumer"
+CONSUMER_JOB_NAME_2="shipping-consumer"
+
+# Observer vars
+OBSERVER_JOB_NAME="ce-kafka-observer"
+OBSERVER_JR_NAME="kafka-observer"
+
+# Mandatory variables provided by user
+IAM_API_KEY=${IAM_API_KEY:-foobar}
+ES_SERVICE_INSTANCE=${ES_SERVICE_INSTANCE:-foobar}
+
+if [ "$IAM_API_KEY" == "foobar" ]; then
+    echo "[ERROR] IAM_API_KEY mandatory env var is not defined."
+    exit 1
+fi
+
+if [ "$ES_SERVICE_INSTANCE" == "foobar" ]; then
+    echo "[ERROR] ES_SERVICE_INSTANCE mandatory env var is not defined."
+    exit 1
+fi
+
+# Verify that the ES service instance exists under the same account
+if ! ibmcloud resource service-instance $ES_SERVICE_INSTANCE -q > /dev/null 2>&1; then
```

Comment:
This script breaks if the ES instance name contains spaces, like '`Event Streams-ki`'

Reply:
```suggestion
if ! ibmcloud resource service-instance "$ES_SERVICE_INSTANCE" -q > /dev/null 2>&1; then
```